modules/update-cvo.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding_updates/how-updates-work.adoc
+
+:_content-type: CONCEPT
+[id="update-cvo_{context}"]
+= The Cluster Version Operator
+
+// adding a poorly written, technically inaccurate skeleton of a module for now, which can be replaced/refined by SMEs as they see fit
```

Comment:
Your following paragraph looks technically accurate to me.  You  could echo the ClusterVersion sentence about ClusterVersion being the API between the CVO and its managers if you wanted, but you could leave it off too.

Personally, the "CVO is constantly comparing ... and reconciling..." bit is the main thing, and install, update, and normal operations are all minor variations on that theme.  But :shrug:, if we cover that in the second sentence instead of the first, like you have now, that's enough for me.

Reply:
I could go either way. If you're fine with how it is, I think it's stronger to start off this "how updates work" page with a sentence about the CVO being the facilitator of updates, and then a clarification that really the CVO does this sort of reconciliation all the time.

But I also understand if that logic sounds backwards and you, or others, want me to emphasize the general reconciliation before diving in to its relevance to updates